server/server.js

```diff
@@ -91,22 +107,11 @@ controller_socket.on('connection', function(socket) {
           angle: angle,
           speed: ctrl.speed  + speedUnit*10
         }
+        sendSound('fire');
       break;
-      case 'left':
-        ctrl.angle -= angleUnit;
-      break;
-      case 'right':
-        worldState.controllers[socket.id].angle += angleUnit;
-      break;
-      case 'forward':
-        speed = worldState.controllers[socket.id].speed + speedUnit;
-        if (speed<maxSpeed) {
-          worldState.controllers[socket.id].speed = speed;
-        }
-      break;
-      case 'back':
-        speed = Math.max(worldState.controllers[socket.id].speed - speedUnit, 0);
-        worldState.controllers[socket.id].speed = speed;
+      case 'orientation':
+        ctrl.beta = 2*Math.PI*Math.floor(Math.abs(beta/10))*10*Math.sign(beta)/360;
```

Comment:
This line is very confusing. You're taking `abs` then multiplying by `sign`, dividing by 10, flooring, then multiplying by 10...
Surely there's a simplified form. Maybe splitting the normalization and the unit conversion into separate lines is a good start.